xml/hardening.xml

```diff
@@ -3445,6 +3445,311 @@ Account Expires: Never</screen>
+    <title>Restricting Local Text Console Logins</title>
+
+    <para>
+     TTY devices provide text mode system access via the console. For desktop
+     systems these are accessed via the local keyboard or in case of server
+     systems via input devices connected to a KVM switch or a remote
+     management card (ILO, DRAC, etc).
+     By default, Linux offers 6 different consoles, that can be switched to
+     via the key-combinations
+     <keycombo><keycap function="alt"/><keycap>F1</keycap></keycombo> to
+     <keycombo><keycap function="alt"/><keycap>F6</keycap></keycombo>, when
+     running in text mode, or
+     <keycombo><keycap function="control"/><keycap function="alt"/><keycap>F1</keycap></keycombo> to
+     <keycombo><keycap function="control"/><keycap function="alt"/><keycap>F6</keycap></keycombo>
+      when running in a graphical session. The
+      associated terminal devices are named <literal>tty1</literal> .. <literal>tty6</literal> accordingly.
+    </para>
+
+    <para>
+     The following steps restrict root access to the first TTY. Even this
+     access method is only meant for emergency access to the system and should
+     never be used for everyday system administration tasks.
+    </para>
+
+    <itemizedlist>
+     <listitem>
+     <para>
+      Ensure that the PAM stack configuration file <filename>/etc/pam.d/login</filename>
+      contains the <literal>pam_securetty</literal> module in the <literal>auth</literal> block:
+     </para>
+     <screen>auth     requisite      pam_nologin.so
+auth     [user_unknown=ignore success=ok ignore=ignore auth_err=die default=bad] pam_securetty.so noconsole
+auth     include        common-auth</screen>
+     </listitem>
+     <listitem>
+      <para>
+       This will include the <literal>pam_securetty</literal> PAM module during the
+       authentication process on local consoles to allow root only to login on
+       tty devices that are listed in the file <filename>/etc/securetty</filename>.
+       Thus to actually limit the tty devices for root you need to remove all
+       entries from this file except one:
+      </para>
+     <screen>#
+# This file contains the device names of tty lines (one per line,
+# without leading /dev/) on which root is allowed to login.
+#
+tty1</screen>
+     </listitem>
+     <listitem>
+      <para>
+       After these changes root logins from non-allowed terminals will be
+       rejected right away without even querying the root password.
+      </para>
+     </listitem>
+    </itemizedlist>
+
+    <important>
+     <para>
+      One might be tempted to add the <literal>pam_securetty</literal> PAM
+      module to the <filename>/etc/pam.d/common-auth</filename> file to make it
+      effective for all PAM services. This would break <literal>su</literal>
+      and <literal>sudo</literal>, however, because these tools would then
+      also reject root authentications.
+     </para>
+    </important>
+
+    <important>
```

Comment:
If possible, try to avoid three consecutive admonitions (2x important and note). This kind of "dilutes" the warning. Could you combine the two important into one?

Reply:
A lot of admonitions may seem strange but these are facts. Combining the two important blocks is technically possible, of course, but they are still two distinct items. So somebody might read the first few sentences and think "doesn't affect me" and skip over the rest of the block.

Reply:
I was more speaking from a documentation and readability perspective. If you think this is too hard then leave it as it is for the time being. :-)